tests/api/test_models.py

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
I think `test_models` can be confusing, can we be more precise like `test_pydantic_models`? or something?